services/src/handlers/spatial_references.rs

```diff
@@ -268,16 +268,29 @@ mod tests {
     #[test]
     fn spec_webmercator() {
         let spec = spatial_reference_specification("EPSG:3857").unwrap();
-        assert_eq!(SpatialReferenceSpecification {
-                name: "WGS 84 / Pseudo-Mercator".to_owned(),
-                spatial_reference: SpatialReference::new(SpatialReferenceAuthority::Epsg, 3857),
-                proj_string: "+proj=merc +a=6378137 +b=6378137 +lat_ts=0 +lon_0=0 +x_0=0 +y_0=0 +k=1 +units=m +nadgrids=@null +wktext +no_defs +type=crs".into(),
-                extent: BoundingBox2D::new_unchecked((-20_037_508.342_789_244, -20_048_966.104_014_6).into(),  (20_037_508.342_789_244, 20_048_966.104_014_594).into()),
-                axis_labels: Some(("Easting".to_owned(), "Northing".to_owned())),
-                axis_order: Some(AxisOrder::EastNorth),
-            },
-            spec
+        assert_eq!(spec.name, "WGS 84 / Pseudo-Mercator");
+        assert_eq!(
+            spec.spatial_reference,
+            SpatialReference::new(SpatialReferenceAuthority::Epsg, 3857)
+        );
+        assert_eq!(
+            spec.proj_string,
+            "+proj=merc +a=6378137 +b=6378137 +lat_ts=0 +lon_0=0 +x_0=0 +y_0=0 +k=1 +units=m +nadgrids=@null +wktext +no_defs +type=crs",
+        );
+        assert!(approx_eq!(
```

Comment:
It would be really great to have an `assert_approx_eq!` macro that does `assert!(approx_eq!(…))`.
For me, it is always risky to forget the assertion :smile:.

Reply:
yes but maybe in another task.